docs/builder/store-promotion.md

```diff
@@ -0,0 +1,109 @@
+
+## Requirements 
+
+To qualify for a free Microsoft Store developer account, you must:
+
+  * own a PWA that is installable, contains all required manifest fields, and implements at least two desktop enhancements. See our [Beginner's Guide to Progressive Web Apps](/home/pwa-intro) for more info.
+  * live in a country or region where the Windows program in Partner Center is offered. See [here](https://learn.microsoft.com/en-us/windows/apps/publish/partner-center/account-types-locations-and-fees#developer-account-and-app-submission-markets) for the full list of countries
+  * have a valid Microsoft Account to use to sign up for the Microsoft Store on Windows developer account
+  * not have an existing Microsoft Store on Windows individual developer/publisher account
+  * use the Store Token to create a Microsoft Store on Windows developer account within 30 calendar days of Microsoft sending you the token, using the same Microsoft Account you used to sign in here
+  * plan to publish an app in the store this calendar year (prior to 12/31/2023 midnight Pacific Standard Time)
+
+To check if your progressive web qualifies and to claim a token, follow the directions below.
+
+## Claiming a Token
+
+Before you can get started claiming a token, make sure you have a valid, public URL for your Progressive Web App.
+
+1. Head to the [PWABuilder.com Store Promotion Page](https://www.pwabuilder.com/freeToken).
+
+2. Enter the URL to your Progressive Web App and hit `Start`.
+
+3. On the report card page, there should be a notification under `Action Items`:
+
+<div class="docs-image">
+   <img src="/assets/builder/store-promotion/token-action-item.png" alt="The token notification on the PWABuilder report card page. ">
+</div>
+
+4. Click `Check Now`.
+
+5. Your PWA will be evaluated to see if it qualifies, if passes all tests, you can continue. Otherwise, you will need to update your PWA to match the requirements. If you see three green check marks at the top right, your PWA is good to go!
+
+6. Click `Sign in with a Microsoft Account to Continue`.
+
+<div class="docs-image">
+   <img src="/assets/builder/store-promotion/tests-passed.png" alt="The success screen on the PWABuilder token promotion page.">
+</div>
+
+7. Sign in with a personal Microsoft account. You can create one for free [here.](https://account.microsoft.com/account)
+
+8. You will need to accept the promotion terms before you can view your token. 
+
+9. Click `View Token Code`.
+
+<div class="docs-image">
+   <img src="/assets/builder/store-promotion/view-token-code.png" alt="PWABuilder promotion page with terms acceptance and token view button.">
+</div>
+
+10. You can now use your token to sign up for an Microsoft Store developer account. See the following section on specific guidance on using your token.
```

Comment:
```suggestion
3. Your PWA will be evaluated to see if it qualifies, if passes all tests, you can continue. Otherwise, you will need to update your PWA to match the requirements. If you see three green check marks at the top right, your PWA is good to go!

4. Click `Sign in with a Microsoft Account to Continue`.

<div class="docs-image">
   <img src="/assets/builder/store-promotion/tests-passed.png" alt="The success screen on the PWABuilder token promotion page.">
</div>

5. Sign in with a personal Microsoft account. You can create one for free [here.](https://account.microsoft.com/account)

6. You will need to accept the promotion terms before you can view your token. 

7. Click `View Token Code`.

<div class="docs-image">
   <img src="/assets/builder/store-promotion/view-token-code.png" alt="PWABuilder promotion page with terms acceptance and token view button.">
</div>

8. You can now use your token to sign up for an Microsoft Store developer account. See the following section on specific guidance on using your token.
```